packages/pluggableWidgets/range-slider-native/src/RangeSlider.tsx

```diff
@@ -85,12 +85,19 @@ export class RangeSlider extends Component<Props, State> {
     }
 
     private onSlide(values: number[]): void {
+        if (values[0] === null || values[1] === null) {
```

Comment:
In what situation would the lib return `null`?

Reply:
When it reaches the maximum it send the max value and another onSLide with null